conduit/plugins/exporters/postgresql/README.md

```diff
@@ -19,26 +19,34 @@ For additional details, refer to the [parsing documentation here](https://pkg.go
 
 The delete-task prunes old transactions according to its configuration. This can be used to limit the size of the database.
 
+### Pruning an Existing Database
+
+The first run after enabling pruning for an existing database must delete all of the older transactions. For many configurations, this process may take many hours or days. While the initial delete is in progress, the transaction table will be locked, and you'll see database errors in the log.
+
+### Cloud Database Optimization
+
+Many cloud databases like RDS will allocate storage as needed. Once allocated, they typically do not deallocate the storage if it is no longer needed. To realize the benefits of data pruning, you should consider migrating to a new database to reset the allocated storage. You can do this by starting a new deployment from scratch.
+
```

Comment:
This is trying to refer to the underlying S3 storage. Maybe allocate is the wrong word here. How about "extend"?

Reply:
I see. Maybe this is a bit too much in the weeds of cloud DB configuration. But I could see an argument for leaving this bread crumb around.

Reply:
I'm also a little hesitant about keeping this note around. 

This only applies to people adding data pruning to an existing database, right?

Reply:
It's a detail I wasn't aware of that probably applies to most indexer runners (today) who may be interested in this feature. In the future, when there are fewer 2.x deployments, it may not be very relevant.